rpm/Makefile

```diff
@@ -0,0 +1,10 @@
+rpm:
+	$(shell /usr/bin/bash ./update-spec-version.sh)
+	spectool -g podman.spec
+	sudo dnf -y builddep podman.spec
```

Comment:
System-wide installs from a Makefile always make me nervous.  Makefiles should be about producing things, not configuration/system management.

Reply:
Clarification: My concern is if somebody unwittingly runs `make package` on their system, they might be very surprised to see a whole bunch of packages being installed as root.